---
bool success
rail_part_affordance_detection/ObjectPartAffordance[] object_part_affordance_list